README.md

```diff
@@ -1,28 +1,31 @@
 # PHP-TUF
```

Comment:
Please ignore changes to this file; they're from a rebase. I thought the PR would automatically redo its diff when rebased against `main`. Sorry!